### PR TITLE
Fix: Functionality to close the BottomSheet

### PR DESCRIPTION
**Pull request by saxenanihal95**
Fixes: #2505

* Removed Button component as it is tightly coupled
to BottomSheet Component

* Exposed isVisible prop so that user can easily
manage BottomSheet Component visibililty

### CODE REVIEW

**Comment by JoeDuncko** [comment]
This looks like a much needed change! Question: is the `onClose` prop being used at all here? Should it be?

**Comment by saxenanihal95** [comment]
@JoeDuncko Yes i was first thinking to provide `onClose` but that is already part of `modalProps`  (`onDismiss`) but that is only supported for ios right now.

File: src/bottomSheet/BottomSheet.js [hunk 1]
```diff
@@ -86,14 +65,16 @@ const styles = StyleSheet.create({
...
+  onClose: () => {},
+  isVisible: false,
 };
 
 BottomSheet.propTypes = {
   list: PropTypes.array,
-  cancelButtonIndex: PropTypes.number,
-  buttonProps: PropTypes.object,
+  modalProps: PropTypes.object,
+  onClose: PropTypes.func,
```
Comment (flyingcircle) [inline src/bottomSheet/BottomSheet.js]: If `onClose` is already provided by `Modal` then I don't think it has to be included here.

**Comment by JoeDuncko** [comment]
Cool, that makes more sense to me now. I think it's a good first step to a much more useful `BottomSheet` component. Thanks so much for getting this in so quickly!

**Pull request merged**